index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./css/styles.css">
+  <link rel="stylesheet" type="text/css" href="./css/projects.css">
+  <link rel="stylesheet" type="text/css" href="./css/about.css">
+  <link rel="stylesheet" type="text/css" href="./css/form.css">
+  <title>Personal-Profolio</title>
+
+</head>
+
+<body>
+  <!-- Headline Section Starts Here -->
+
+  <header class="navigation">
+    <div class="logo">
+      <h2>My logo</h2>
```

Comment:
- [ ] Please change the "logo" to an `a` tag instead of an `h2` as instructed by the previous reviewer. As this can be used later as a link to the home page.